packages/react-icon-provider/package.json

```
@@ -0,0 +1,50 @@
+{
+  "name": "@fluentui/react-icon-provider",
+  "version": "0.1.0",
+  "description": "Package for applying icon overrides to new svg icons",
+  "private": true,
```

Would like @xugao's input on this too, but I think we should start the package at 1.0.0-beta and go ahead and publish it.
```suggestion
  "version": "1.0.0-beta.1",
  "description": "Package for applying icon overrides to Fluent UI React SVG icons",
```